lib/fastlane/plugin/wpmreleasetoolkit/models/firebase_device.rb

```diff
@@ -0,0 +1,49 @@
+module Fastlane
+  class FirebaseDevice
+    attr_reader :model, :version, :locale, :orientation
```

Comment:
Yeah, I worried about this for a moment as well – IMHO if that happens at some point we can address it then? I didn't want to bury it too deeply since it's a pain to reference a deeply-nested model object?

Reply:
Agreed. I think we could also keep the idea in the back of our head for later to introduce a `module WPMRT` namespace for models at some point.

I worried about it for [this old WIP as well](https://github.com/wordpress-mobile/release-toolkit/pull/296/files#diff-bdb20f98842eb1bfe87ab2b48e33f15dccb6779e9509cf514759d8bbaa605378), which might be even more at risk of clashing with something provided by fastlane given its more generic name… so maybe at that point create that `WPMRT` namespace then. Given yours is not introduced as public API (iinm), changing the internal namespace and thus fully-qualified name of that class later would not be a breaking change (unlike the `LocaleHelper` PR which is planned to be exposed publicly)

Reply:
> changing the internal namespace and thus fully-qualified name of that class later would not be a breaking change

Yeah exactly – this was what had me thinking it'd make sense to leave it for the moment 🙂

Reply:
One thing we could consider at least doing though: put all the Firebase-related models under a `models/firebase` subdirectory, and use `Fastlane::Firebase::Device` instead of `Fastlane::FirebaseDevice` (and similar for the 2 others). (I think Rubymine can do such a refactor automatically accross the codebase…?)

Since those are models that all go together hand in hand that would make it clear and neat, and would make us ready for if we add more, unrelated model files for other unrelated things in the future, avoinding to put everything in the same bowl.

Reply:
Note that:
 - The `models` will be exposed to the public now thanks to https://github.com/wordpress-mobile/release-toolkit/pull/355/files#diff-ea7a49d236e63ebdbd0e50b21527ce2fa9887c7666ce0145eb9b6b6d1d393542R7 
 - Which means it will be a breaking change to switch their API
 - Though it could be a good thing to make them public if we end up changing the API of this action to accept multiple devices (like the FTL API does, and as we'll need it for various cases in our client apps in practice) and want the action's API to accept an array of `Firebase::Device`… that again will need a breaking change in API in the next iteration tho